bin/test/i18n/resources/apps/musiclab_libraries/test_sync_in.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../../../../test_helper'
+require_relative '../../../../../i18n/resources/apps/musiclab_libraries/sync_in'
+
+describe I18n::Resources::Apps::MusiclabLibraries::SyncIn do
+  let(:described_class) {I18n::Resources::Apps::MusiclabLibraries::SyncIn}
+  let(:described_instance) {described_class.new}
+
+  around do |test|
+    FakeFS.with_fresh {test.call}
+  end
+
+  it 'inherits from I18n::Utils::SyncInBase' do
+    assert_equal I18n::Utils::SyncInBase, described_class.superclass
+  end
+
+  describe '#process' do
+    let(:process) {described_instance.process}
+
+    let(:i18n_source_file_path_intro) {CDO.dir('i18n/locales/source/musiclab_libraries/music-library-intro2024.json')}
+    let(:i18n_source_file_path_launch_v2) {CDO.dir('i18n/locales/source/musiclab_libraries/music-library-launch2024.json')}
+
+    it 'prepares the i18n source file' do
+      described_instance.expects(:get_translation_strings).returns({'lady_gaga_born_this_way/vocals_verse_3': 'Verse 3'}).twice
```

Comment:
Can we do this with a sound in one of our default packs, rather than a commercial artist one?

Reply:
Sorry this is just an overly detailed example, this stub could be anything. I can put something more generic to make that more clear.

Reply:
Looks like this is stubbing the `get_translation_strings` function right? Any way we can also test the logic inside that function to make sure we're grabbing the right values and computing IDs correctly?

Reply:
Yep that's right, and a good callout -- I was torn on whether it was simple enough to skip testing the guts of the function...I'll try and circle back and add a couple if I have a bit of time.